plumbing/util.go

```diff
@@ -230,7 +222,9 @@ func CacheDir() string {
 		}
 	}
 
-	MkdirAsNeeded(cacheDir)
+	if err := os.MkdirAll(cacheDir, 0777); err != nil {
+		log.Errorf("%v", err)
+	}
```

Comment:
For each of the three `os.MkdirAll`s in this file, how fantastically do we expect the rest of the invocation to blow up if we reach errors? Is it worth plumbing them through to their callers?

Reply:
Opened ch-2567